cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ScalarFunctionsTest.scala

```diff
@@ -186,71 +210,73 @@ class ScalarFunctionsTest extends DocumentingTest {
       }
     }
     section("size()", "functions-size") {
-      p(
-        """`size()` returns the number of elements in a list.""".stripMargin)
-      function("size(list)", "An Integer.", ("list", "An expression that returns a list."))
+      p("The function `size()` returns the number of elements in a list.")
+      function("size(list)",
+        "An Integer.",
+        ("list", "An expression that returns a list."))
       considerations("`size(null)` returns `null`.")
-      query(
-        """RETURN size(['Alice', 'Bob'])""".stripMargin, ResultAssertions((r) => {
+      query("RETURN size(['Alice', 'Bob'])",
+      ResultAssertions((r) => {
           r.toList should equal(List(Map("size(['Alice', 'Bob'])" -> 2)))
         })) {
         resultTable()
         p("The number of elements in the list is returned.")
       }
     }
     section("size() applied to pattern expression", "functions-size-of-pattern-expression") {
-      p(
-        """This is the same `size()` method as described above, but instead of passing in a list directly, a pattern expression can be provided that can be used in a match query to provide a new set of results.
-          |These results are a _list_ of paths.
-          |The size of the result is calculated, not the length of the expression itself.""".stripMargin)
-      function("size(pattern expression)", ("pattern expression", "A pattern expression that returns a list."))
-      query(
-        """MATCH (a)
-          |WHERE a.name = 'Alice'
-          |RETURN size((a)-->()-->()) AS fof""".stripMargin, ResultAssertions((r) => {
+      p("""This is the same function `size()` as described above, but instead of passing in a list directly, a pattern expression can be provided that can be used in a match query to provide a new set of results.
```

Comment:
```suggestion
      p("""This is the same function `size()` as described above, but you pass in a pattern expression, instead of a list, which is used in a match query to provide a new set of results.
```

Reply:
A suggestion to remove the duplicate use of "provide".